### ISSUE TITLE
Improvement: Calibration of Nozzle Opening – Increase only scheme not as user-friendly as intended

### ISSUE THREAD

**Issue by Wookbert** [celsworthy/AutoMaker#15]
Nozzle Open Calibration allows the user only to open, but not to close the valve, in other words decrease the value before hitting „Present“ (= Accept)?

The typical newbie user will be impatient and won’t understand that it may take a second or two for the filament to pass the needle valve if it is opened by a very small degree. Instead one clicks hasty „Not present“ which leads to the valve being open, when the printer thinks it is closed –> oozing.

If an experienced user notice his mistake, his only option is to cancel the calibration and start over — a time-consuming process with all the cool down and re-heat. The numeric entry field can be edited, yet the entry doesn’t get accepted. Huh?

Bottom line: the „Not Present“ and „Present“ button might be intended to be fool-proof, but in truth they are not. I'd suggest to rephrase the user instructions, and have three buttons (Open, Close, Accept) and numeric feedback, so follow the same scheme you already have in Nozzle Height Calibration (where you can Raise or Lower the head, so go back and forth, until it fits).
